### PR TITLE
eventcmd.sh: Use XDG paths

### PR DESCRIPTION
**Pull request by RichiH**
Arguably, this should be in $XDG_CACHE_HOME instead...

### CODE REVIEW

File: contrib/eventcmd-examples/eventcmd.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ case "$1" in
 #	songstart)
 #		echo 'naughty.notify({title = "pianobar", text = "Now playing: ' "$title" ' by ' "$artist" '"})' | awesome-client -
 
-#		echo "$title -- $artist" > $HOME/.config/pianobar/nowplaying
+#		echo "$title -- $artist" > $XDG_CONFIG_HOME/pianobar/nowplaying
```
Comment (rabin-io) [inline contrib/eventcmd-examples/eventcmd.sh]: If so, why not , 

    echo "${title} -- ${artist}" > "${XDG_CONFIG_HOME}/pianobar/nowplaying"

Comment (RichiH) [inline contrib/eventcmd-examples/eventcmd.sh]: You are right, see new patch in this PR.

**Comment by PromyLOPh** [comment]
The variable can be (and probably is for a lot of people)
undefined/empty.

**Comment by rabin-io** [comment]
in that case you case use the old path as a default value 

     echo ${XDG_HOME_CONFIG:-${HOME}/.config}

**Pull request closed without merging**